### PR TITLE
assisted-day2: added presubmit test to agent

### PR DESCRIPTION
**Pull request by danielerez**
Added e2e-metal-assisted-day2 to assisted-installer-agent

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml [hunk 1]
```diff
@@ -73,6 +73,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml:77]: just to make sure
do you want it to always run on every PR of agent? or run only if someone does ``/test e2e-metal-assisted-day2``?
if the former, leave as is. if the later, change to ``always_run: false``

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml (hunk 1 above)
Comment (danielerez) [reply ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml:77]: hmm, marked it as optional, should it been auto generated accordingly? :/

Comment (osherdp) [reply ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml:77]: so optional meaning not required for merging the PR
but it's always auto-generatedso that the job is being triggered on every PR

**Review by osherdp — APPROVED** [review]

**Pull request merged**